services/net/filemonitor/FilemonitorAction.cs

```diff
@@ -470,6 +470,8 @@ private async Task GetXmlArticlesAsync(string dir, IIngestActionManager manager,
                         await ImportArticleAsync(manager, item);
                     }
                 }
+                // This ingest has just completed running for one content item.
+                await manager.UpdateIngestStateAsync(manager.Ingest.FailedAttempts);
```

Comment:
Move to only update status on success, as the error handler already updates the status.